src/build/env.py

```diff
@@ -42,9 +42,20 @@ def install(self, requirements):  # type: (Iterable[str]) -> None
 
 
 class IsolatedEnvBuilder(object):
-    def __init__(self):  # type: () -> None
-        """Builder object for isolated environment."""
+    def __init__(self, silent=True):  # type: (bool) -> None
+        """
+        Builder object for isolated environment
+
+        :param silent: Silence the output
+        """
         self._path = None  # type: Optional[str]
+        self._silent = silent
+        self._outfd = sys.stdout  # type: Union[int, TextIO, IO[str]]
+        if silent:
+            if sys.version_info[0] == 2:
+                self._outfd = open(os.devnull, 'w')
```

Comment:
How does this work on Windows?

Reply:
It should.

https://docs.python.org/3/library/os.html#os.devnull